Guide.docc/IncrementalAdoption.md

```diff
@@ -149,6 +149,33 @@ class UIStyler {
 Combining static and dynamic isolation can be a powerful tool to keep the
 scope of changes gradual.
 
+### Manual MainActor Isolation
```

Comment:
Something like `Changing execution context to the MainActor` maybe?

I know you mean manual in terms of "not just the annotation which makes it work" but I wonder if it's a bit unusual to refer to calling async funcs with right isolation as "manual" 🤔